src/features/navbar/navSlice.js

```diff
@@ -0,0 +1,16 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+export const navSlice = createSlice({

```

Comment:
For such small slices, you can combine such small slices like nav and alert into a common app slice. Not very sure about this, please research about this before going ahead with this.